src/game/boe.actions.cpp

```diff
@@ -2114,29 +2113,19 @@ bool handle_keystroke(const sf::Event& event){
 }
 
 bool handle_scroll(const sf::Event& event) {
-	rectangle status_panel_rect = {0,0,144,271}, text_panel_rect = {0,0,138,271};
```

Comment:
It's not immediately clear why these were removed, but... I guess they're now set as the control rect of the two scrollbars?

Reply:
Yes. And these magic numbers were not even used as-is, they were "adjusted" using .offset before being used.

And this calculation was performed per every mousewheel event processed.

So I removed this and instead use the "already adjusted" values as event catching rects.